app/views/coronavirus_landing_page/components/shared/_accordion_sections.html.erb

```diff
@@ -34,10 +34,8 @@
           event_name: 'select_content',
           type: 'accordion',
           text: section["title"],
-          index: index + 1,
+          index: index,
```

Comment:
I can't see the reference to the track click module in this file - is it somewhere in a parent template?

Reply:
Sorry, fixed now